### DISCUSSION TITLE
OSickGear updating package cryptography...

### DISCUSSION THREAD

**Discussion by scarng63** [OneCDOnly/sherpa#254] [General]
Everytime I "Show List" I get a message to "Python package updates available — [Update Now]"  When I do the update I get the following error message:
Updating package "cryptography" version 38.0.4 to 39.0.0: **Can not perform a '--user' install. User site-packages are not visible in this virtualenv.**
I've run "sherpa upgrade OSickGear" still keep getting the message.
Anyone else having this issue?

**Comment by OneCDOnly** [comment]
Hi @scarng63 :)

If you'd like to test a possible fix, can you please edit your `osickgear.sh` file and change line 50 that reads:
```
readonly ALLOW_ACCESS_TO_SYS_PACKAGES=false
```
... to read as: 
```
readonly ALLOW_ACCESS_TO_SYS_PACKAGES=true
```
... then restart the QPKG? 
```
/etc/init.d/osickgear.sh restart
```
I can't promise this will fix things, as it's not possible to upgrade site packages from within the virtualenv, but it may give SickGear a better idea as to what's going on.

**Comment by scarng63** [comment]
applied the change and did the restart.   Same issue with cryptography...

On Tue, Jan 3, 2023 at 2:04 PM One CD ***@***.***> wrote:

> Hi @scarng63 <https://github.com/scarng63> :)
>
> If you'd like to test a possible fix, can you please edit your
> osickgear.sh file and change the line that reads:
>
> readonly ALLOW_ACCESS_TO_SYS_PACKAGES=false
>
> ... to read as:
>
> readonly ALLOW_ACCESS_TO_SYS_PACKAGES=true
>
> ... then restart the QPKG?
>
> /etc/init.d/osickgear.sh restart
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OneCDOnly/sherpa/discussions/254#discussioncomment-4584870>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXJ3V2WWIRK3V6NJPPLWAPLWQRZ5FANCNFSM6AAAAAATPVWSBU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
~Jules Edwards
803-960-0157

**Comment by OneCDOnly** [comment]
It looks like you'll need to disable internal git updating in your Sickgear application. Updates and dependencies are handled by the service script anyway.

**Comment by scarng63** [comment]
How to disable the internal git updating in the SickGear application?

On Tue, Jan 3, 2023 at 6:16 PM One CD ***@***.***> wrote:

> It looks like you'll need to disable internal git updating in your
> Sickgear application. Updates and dependencies are handled by the service
> script anyway.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OneCDOnly/sherpa/discussions/254#discussioncomment-4586479>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXJ3V2UCAH2SMC54S2KHBSLWQSXOVANCNFSM6AAAAAATPVWSBU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
~Jules Edwards
803-960-0157

**Reply by OneCDOnly** [reply]
The settings for this can be seen in the SickGear UI. 

Go to: Settings -> General -> Misc tab -> scroll to bottom and view the Updates section. Ensure all tickboxes are unticked.

**Comment by scarng63** [comment]
Done, thank you.

On Wed, Jan 4, 2023 at 1:58 PM One CD ***@***.***> wrote:

> The settings for this can be seen in the SickGear UI.
>
> Go to: Settings -> General -> Misc tab -> scroll to bottom and view the
> Updates section. Ensure all tickboxes are unticked.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OneCDOnly/sherpa/discussions/254#discussioncomment-4594779>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXJ3V2RN5BX6ZTVMO42SMJLWQXB7FANCNFSM6AAAAAATPVWSBU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
~Jules Edwards
803-960-0157
